index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
+
+  <title>My Portfolio</title>
+</head>
+<body>
+<header>
+  <nav class="d-flex header">
+    <a href="#" class="logo  nav-link"><span>My Logo</span></a>
+    <span class="material-icons nav-link" >menu</span>
+  </nav>
+</header>
+
+  <section class="article">
+    <h1 class="d-flex information">I'm Mohamed Badr <br> glad to see <br> you!</h1>
+    <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! <br>If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <h2 class="contact">LET'S CONNECT</h2>
+    <ul class="d-flex  icon-links">
+      <li><a href="https://github.com/mbdesigns1989"><img src="./img/github.png" alt=""></a></li>
+      <li><a href="https://www.linkedin.com/in/mohamed-badr-27b26a212/"><img src="./img/icon1.png" alt=""></a></li>
+      <li><a href="https://twitter.com/mohamed66083152"><img src="./img/icon2.png" alt=""></a></li>
+
+      <!-- <li><a href="https://www.linkedin.com/in/mohamed-badr-27b26a212/"><img src="./img/icon1.png" alt=""></a></li>
+      <li><a href="https://twitter.com/mohamed66083152"><img src="./img/twitter.png" alt=""></a></li> -->
+    </ul>
+  </section>
```

Comment:
- [ ] It would be great if this `<section>` is aligned with flexbox. This will help for responsiveness for the next steps in your project. 🐶